dist/mapdc.js

```diff
@@ -43765,20 +43781,13 @@ function segmentEach(geojson, callback) {
 
         // Generate 2-vertex line segments
         var previousCoords;
-        var previousFeatureIndex = 0;
-        var previousMultiIndex = 0;
-        var prevGeomIndex = 0;
-        if (coordEach(feature, function (currentCoord, coordIndex, featureIndexCoord, multiPartIndexCoord, geometryIndex) {
+        if (coordEach(feature, function (currentCoord, coordIndex, featureIndexCoord, mutliPartIndexCoord, geometryIndex) {
             // Simulating a meta.coordReduce() since `reduce` operations cannot be stopped by returning `false`
-            if (previousCoords === undefined || featureIndex > previousFeatureIndex || multiPartIndexCoord > previousMultiIndex || geometryIndex > prevGeomIndex) {
+            if (previousCoords === undefined) {
```

Comment:
Actual difference

Reply:
@biovisualize, do you know which PR relates to this part?